.vscode/settings.json

```diff
@@ -22,5 +22,6 @@
     "--profile",
     "black"
   ],
-  "python.testing.pytestEnabled": true
+  "python.testing.pytestEnabled": true,
+  "python.pythonPath": "/home/zuppif/miniconda3/bin/python"
```

Comment:
Can you unstage this local dev-env specific line? Thanks!

Reply:
yes